packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1796,7 +1796,7 @@ function updateHostRoot(
   }
 
   const nextProps = workInProgress.pendingProps;
-  const prevState = workInProgress.memoizedState;
+  const prevState: RootState = workInProgress.memoizedState;
```

Comment:
Drive-by when I looked through existing usages of memoized state of HostRoot. memoizedState is `any` by default so this just increases Flow coverage.